command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
As much as it pains me to say it, I can't come up with a good reason to disagree with you regarding putting the runtime cli spec in an opencontainers/runtime-client-spec repo. An alternative would be to put it into a submodule/sub-project.